fluid/PaddleCV/image_classification/README_ngraph.md

```diff
@@ -0,0 +1,66 @@
+
+# PaddlePaddle inference and training script
+This directory contains model configuration and tool used to run the PaddlePaddle + NGraph for a local training and inference.
+
+# How to build PaddlePaddle framework with NGraph engine
+In order to build the PaddlePaddle + NGraph engine and run proper script follow up a few steps:
+1. build the PaddlePaddle project
+2. download pre-trained model data
+3. set env exports for nGraph and OMP
+5. run the inference/training script
+
+Curently supported models:
+* ResNet50 (inference and training).
+
+Short description of aforementioned steps:
+
+## 1. Build paddle
+Do it as you usually do. In case you never did it, here are instructions:
+```
+git clone https://github.com/PaddlePaddle/Paddle.git
+cd Paddle
+mkdir build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE=Release -DWITH_DOC=OFF -DWITH_GPU=OFF -DWITH_DISTRIBUTE=OFF -DWITH_MKLDNN=ON -DWITH_MKL=ON -DWITH_GOLANG=OFF -DWITH_SWIG_PY=ON -DWITH_STYLE_CHECK=OFF -DWITH_TESTING=ON -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DWITH_PROFILER=OFF -DWITH_NGRAPH=ON
+```
+## 2. Download pre-trained model:
+In order to download model, go to data/ILSVRC2012/ directory and run download_resnet50.sh script:
+```
+$ cd data/ILSVRC2012/
+$ ./download_resnet50.sh
+```
+
+## 3. Set env exports for nGraph and OMP
+Set the following exports needed for running nGraph:
+```
+export FLAGS_use_ngraph=true
+export KMP_AFFINITY=granularity=fine,compact,1,0
+export KMP_BLOCKTIME=1
+export OMP_NUM_THREADS=<num_cpu_cores>
+```
+
+## 4. How the benchmark script might be run.
```

Comment:
Build paddle should just only link the paddle's doc. 

Or use this https://github.com/PaddlePaddle/models/tree/develop/fluid/PaddleCV/image_classification#installation

This readme should focus on the difference of the original readme in this folder or just highlight `DWITH_MKLDNN=ON -DWITH_MKL=ON `, and what is must be done, for example requirement of machine? at least AVX2.